google-maps/ios/Plugin/Circle.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import Capacitor
+
+public struct Circle {
+    let center: LatLng
+    let radius: Double
+    let strokeWidth: CGFloat
+    let strokeColor: UIColor
+    let fillColor: UIColor
+    let tappable: Bool?
+    let title: String?
+    let zIndex: Int32
+    let tag: String?
+
+    init(fromJSObject: JSObject) throws {
```

Comment:
Minor nitpick that won't block anything, but `init(from jsObject: JSObject)` is a little more post Swift 3 swifty.